### DISCUSSION TITLE
Fog of War Support

### DISCUSSION THREAD

**Discussion by MeusRex** [MeusRex/pf2e-chex#4] [Ideas]
If token vision is enabled, players should only know about discovered cells. (E.g. cells they have viewed with a token.)

- [ ] Only show HUD for seen
- [x] Apply fog to overlay

**Comment by federicoacker** [comment]
To add to this, I haven't checked yet but is there a way to make it so that visited hexes get automatically flagged as mapped and then subsequently "cleared" when the token moves out of the hex?

**Reply by MeusRex** [reply]
I'll need to think about this. I though the cleared stood for "cleared by the kingdom" since Kingmaker has a clear a hex (to prepare it for building.)
Auto clearing it would feel strange. Maybe a setting in the options.

**Reply by federicoacker** [reply]
Oh my bad, I thought cleared meant something like "party has cleared the hex, they have done everything they had to do here"
